frontend/awx/resources/templates/WorkflowVisualizer/types.ts

```diff
@@ -8,3 +8,14 @@ export type GraphNode = Omit<NodeModel, 'data'> & {
 export type LayoutNode = WorkflowNode & {
   runAfterTasks?: string[];
 };
+
+export interface CustomEdgeProps {
+  element: GraphElement<
+    ElementModel,
+    {
+      tag: string;
```

Comment:
Does topology support the tag being a react element?  If so, perhaps we could pass it a context menu instead of just a string.

Reply:
It currently doesn't support `ReactNode`. I brought this up with the PatternFly team and they will look into making this prop a `ReactNode` instead of a `string`.